hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -4412,6 +4412,9 @@ public static boolean areNodeLabelsEnabled(
   public static final String AM_DEFAULT_NODE_LABEL =
       RM_NODE_LABELS_PREFIX + "am.default-node-label-expression";
 
+  public static final String ALLOWED_AM_NON_EXCLUSIVE_ALLOCATION =
+      RM_NODE_LABELS_PREFIX + "am.allowed-non-exclusive-allocation";
+
```

Comment:
Based on the actual property name I think AM_ALLOWED_NON_EXCLUSIVE_ALLOCATION would be better. Or simply using am.allow-non-exclusive-allocation might sound more natural.

Reply:
Got it.